viewer/packages/360-images/src/Image360Action.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+/**
+ * 360 image action to be used by the CogniteViewer.canDo360Action and do360Action
+ * @beta
+ */
+export enum Image360Action {
+  Forward, // When inside 360 image, forwards on the history list
+  Backward, // When inside 360 image, backwards on the history list
+  Enter, // When outside 360 image, go to current in the history list (where you exit from)
+  Exit // When inside 360, exit.
+}
```

Comment:
```suggestion
export enum Image360Action {
  /**
  * When inside 360 image, forwards on the history list.
  */
  Forward,
  /**
  * When inside 360 image, backwards on the history list.
  */
  Backward,
  /**
  * When outside 360 image, go to current in the history list (where you exit from).
  */
  Enter,
  /**
  * When inside 360, exit.
  */
  Exit
}
```

Reply:
This suggestion will removed undocumented tag in reveal.api.md file

Reply:
Fixed